lib/design/status/success_page.dart

```diff
@@ -1,39 +1,35 @@
 import 'package:flutter/material.dart';
 import 'package:get/get.dart';
+import 'package:get/get.dart' as Get;
 import 'package:hypha_wallet/design/background/hypha_half_background.dart';
 import 'package:hypha_wallet/design/background/hypha_page_background.dart';
 import 'package:hypha_wallet/design/bottom_component/hypha_safe_bottom_navigation_bar.dart';
+import 'package:hypha_wallet/design/buttons/button_type.dart';
 import 'package:hypha_wallet/design/buttons/hypha_app_button.dart';
 import 'package:hypha_wallet/design/hypha_colors.dart';
 import 'package:hypha_wallet/design/themes/extensions/theme_extension_provider.dart';
 import 'package:hypha_wallet/ui/bottom_navigation/hypha_bottom_navigation.dart';
+import 'package:hypha_wallet/ui/proposals/details/proposal_details_page.dart';
 
-enum SignSuccessTransactionType {
-  approved('Approved');
+enum SuccessType {
+  approved('Approved'),
+  published('published');
 
-  const SignSuccessTransactionType(this.value);
```

Comment:
Why have these pages be renamed? They're still about signing, they should keep the same, more descriptive name, they have before.